src/target/drivers/display/8080/320x240x16.c

```diff
@@ -35,17 +37,23 @@ void lcd_set_pos(unsigned int x0, unsigned int y0)
     //printf("lcd_set_pos: %d, %d\n", x0, y0);
 }
 
-#define LCDTYPE_HX8347  1
-#define LCDTYPE_ILI9341 2
-#define LCDTYPE_UNKNOWN 0
+#define LCDTYPE_HX8347  0x01
```

Comment:
can we move this define to some .h file as target_defs.h is referencing it.

Reply:
there's not really a good place to put it right now.  Putting it in a common file would just be clutter  to any other tx, and the target_defs is loaded for virtually every c file.  We don't have a define to tell us which display is used, used there isn't a way to set these things based on a header.  Maybe when I rework the target_defs.h or build system it will be easier to define a proper way to handle this without needing to have redundant information.  Ideally screen-size should be set be defining the display-driver, but that too isn't possible right now.